charts/incubator/ai-server/values.yaml

```diff
@@ -1,11 +1,24 @@
 image:
   pullPolicy: IfNotPresent
-  repository: codeproject/senseai-server
+  repository: codeproject/ai-server
   tag: cpu-2.5.1@sha256:4ac7e0ace01abdb26e6d1876011a95f9f294352abd23936e4c6040f23582ed9f
-gpuimage:
+gpu12Image:
   pullPolicy: IfNotPresent
-  repository: codeproject/senseai-server
+  repository: codeproject/ai-server
   tag: cuda12_2-2.5.1@sha256:c846c67adb140912ff344a7ba57b3498019d18dfeaf70a32ab68ff389db74efd
+gpu11Image:
+  pullPolicy: IfNotPresent
+  repository: codeproject/ai-server
+  tag: cuda11_7-2.5.1@sha256:b48a17af31d10b839baaa07495449bb185493e0d90d88520bd6d6339e6a168cf
+imageSelector: image
```

Comment:
needs to be added within the workload, example [here](https://github.com/truecharts/charts/blob/5c7b7cf0d8f7e53c477249ee05e3917bd30ea2a1/charts/stable/minecraft-java/values.yaml#L111).
```suggestion
```